packages/types/src/lib/interfaces.ts

```diff
@@ -1610,8 +1610,10 @@ export interface RetryTolerance {
   maxRetryCount?: number;
 }
 
-export interface MintCapacityCreditsPerDayContext {
-  requestsPerDay: number;
+export interface MintCapacityCreditsContext {
+  requestsPerDay?: number;
+  requestsPerSecond?: number;
+  requestsPerKilosecond?: number;
   daysUntilUTCMidnightExpiration: number;
 }
```

Comment:
This could be split into the superposition of three interfaces, each one having just one of those values. That would provide better safety and make clear for a user that they must provide ONE value, not several or none